Update trouble_network_config_bm.adoc

jgato opened the pull request:
clarifications and context about where/how find the symptoms. working with @swopebe

jgato committed:
Update trouble_network_config_bm.adoc

clarifications and context about where/how find the symptoms.
working with @swopebe

swopebe commented on clusters/support_troubleshooting/trouble_network_config_bm.adoc:
```diff
@@ -1,29 +1,29 @@
 [#troubleshooting-network-config-fail]
-= Troubleshooting adding day-two nodes to an existing cluster fails with pending user action
+= Troubleshooting: Adding day-two nodes to an existing cluster fails with pending user action
```
Ok, I see the issue here. I agree we need to use a colon, the problem is all the troubleshooting files are not set up this way, and we keep the same format in the files. (This sets up user expectation and makes it easier for any writer to pick up a JIRA and write one)

I will change the other files to match what you changed back.

jgato replied:
so... should I change it back to not use semicolon, or you will change the other files?

swopebe replied:
just leave it, I will make the other files look like this.

swopebe committed:
Update trouble_network_config_bm.adoc

1. Console is just console, we can call it integrated console because it is OCP with MCE. We cannot call it RHACM console because this is MCE doc that does not account for the extra RHACM pieces beyond MCE. We need to write it as I have it here.

2. We only use proper abbreviations, we don't usually abbreviate resource names. We also set them off with code.

3. We do not need to say, "This time" as the Troubleshooting topic is guiding the user to do something different.

swopebe commented on clusters/support_troubleshooting/trouble_network_config_bm.adoc:
```diff
@@ -1,29 +1,29 @@
+The configuration of the network is failing. In the console, you see a `Pending` user action. In the description, you can see it failing on the rebooting step.
 
-The error message about failing is not very accurate, since the agent cannot report information.
+The error message about failing is not very accurate. Since the agent that is running in the installing host cannot report information.
```
These are now not complete sentences. We need the structure to remain as I have written it. Complete sentence: Subject and an action with the phrase.

swopebe committed:
Update trouble_network_config_bm.adoc

swopebe committed:
Update trouble_network_config_bm.adoc

swopebe commented on clusters/support_troubleshooting/trouble_network_config_bm.adoc:
```diff
@@ -1,29 +1,29 @@
+After the Discover phase, the host reboots to continue the installation, but it cannot configure the network. Check for the following symptoms and messages:
 
-* In the console, check for `Pending` user action on the adding node, with the `Rebooting` indicator:
+* From the managed cluster in the integrated console, check for `Pending` user action on the adding node, with the `Rebooting` indicator:
```
These need to also be complete sentences,, as they already have a colon to introduce the next thing.

swopebe commented on clusters/support_troubleshooting/trouble_network_config_bm.adoc:
```diff
@@ -1,29 +1,29 @@
  ** `/sysroot/etc/sysconfig/network-scripts/` 
 
-In the console, check the failing host for the following messages. You can also use `journalctl` to see the log messages:
+* From the terminal of the installing host, check the failing host for the following messages. You can use `journalctl` to see the log messages:
```
How we normally write this is: Run the following command in the terminal to see the log messages:

swopebe replied:
Here is an example (older version) of how we write procedures in the official Red Hat documentation:

https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.6/html-single/multicluster_engine/index#troubleshooting-klusterlet-with-degraded-conditions-mce

swopebe replied:
We don't always even say terminal because you run the command in the terminal, but I am fine to leave when it helps to clarify. Is it better to say:

In the terminal, run the command.

jgato committed:
Update trouble_network_config_bm.adoc

last clarifications on terminology
